### PR TITLE
Finish remove user

### PR DESCRIPTION
**Pull request by trantube94**
I have created a pull request, check it please!

### CODE REVIEW

**Comment by hienbx94** [comment]
Ok

**Comment by ttuan** [comment]
OK

**Comment by ThuBM** [comment]
OK

File: app/views/admin/courses/show.html.erb [hunk 1]
```diff
@@ -34,8 +34,13 @@
 <h2><%= t "course.list_users" %></h2>
 <% @users.each do |user| %>
```
Comment (chuanhtuan) [inline app/views/admin/courses/show.html.erb]: em nên for each theo user_course

File: app/models/user_course.rb [hunk 2]
```diff
@@ -1,6 +1,9 @@
 class UserCourse < ActiveRecord::Base
   belongs_to :course
   belongs_to :user
+  has_many :user_subjects, dependent: :destroy
 
   scope :course_active_by_user, ->{where active: true}
+  scope :get_user_course, ->user_id, course_id{where "user_id = ?
```
Comment (chuanhtuan) [inline app/models/user_course.rb]: không cần dùng cái scope này

File: config/routes.rb [hunk 3]
```diff
@@ -11,6 +11,10 @@
     root "dashboards#index"
     resources :users
     resources :subjects
-    resources :courses
+    resources :courses do 
+      member do
```
Comment (chuanhtuan) [inline config/routes.rb]: remove member

File: config/routes.rb [hunk 4]
```diff
@@ -11,6 +11,8 @@
     root "dashboards#index"
     resources :users
     resources :subjects
-    resources :courses
+    resources :courses do 
+      resources :user_courses
```
Comment (chuanhtuan) [inline config/routes.rb]: only destroy

**Pull request merged**